project/Dependencies.scala

```diff
@@ -40,4 +42,20 @@ object Dependencies {
   lazy val jGit = "org.eclipse.jgit" % "org.eclipse.jgit" % "5.6.0.201912101111-r"
 
   lazy val typesafeConfig = "com.typesafe" % "config" % "1.4.0"
+
+  val akkaVersion = "2.5.26"
+  val akkaHttpVersion = "10.1.9"
+
+  val akka =
+    Seq(
+      "com.typesafe.akka" %% "akka-actor" % akkaVersion force,
+      "com.typesafe.akka" %% "akka-stream" % akkaVersion force,
+      "com.typesafe.akka" %% "akka-slf4j" % akkaVersion force,
```

Comment:
Why do you need to force the akka version?

Reply:
It seems we don't need it. Removed the `force`